src/Host/Client/Impl/rtvs/R/util.R

```diff
@@ -283,36 +290,35 @@ query_reload_autosave <- function() {
         });
 
         if (loaded) {
-            message(sprintf(locstr('rtvs_LoadedWorkspace'), autosave_filename));
+            loc_warning('rtvs_LoadedWorkspace', autosave_filename);
             # If we loaded the file successfully, it's safe to delete it - this session contains the reloaded
             # state now, and if there's another disconnect, it will be autosaved again.
             return(TRUE);
         } else {
-            warning(sprintf(locstr('rtvs_FailedToLoadWorkspace'), autosave_filename), call. = FALSE, immediate. = TRUE);
+            loc_warning('rtvs_FailedToLoadWorkspace', autosave_filename);
             return(FALSE);
         }
     } else {
-        msg <- locstr('rtvs_ConfirmDeleteWorkspace');
-        res <- winDialog('yesno', sprintf(msg, autosave_filename));
-        return(identical(res, 'YES'));
+        res <-loc_askYesNo('rtvs_ConfirmDeleteWorkspace', autosave_filename)[[1]];
+        return(identical(res, 'Y'));
     }
 }
 
 save_state <- function() {
-    # This function runs when client is already disconnected, so locstr() cannot be used, since it requires
+    # This function runs when client is already disconnected, so loc_message() cannot be used, since it requires
     # a connected client to provide translated strings. However, since messages are not user-visible and are
     # here for logging purposes only, they don't need to be localized in the first place.
-    message(sprintf('Autosaving workspace to image "%s" ...', autosave_filename), appendLF = FALSE);
+    #message(sprintf('Autosaving workspace to image "%s" ...', autosave_filename), appendLF = FALSE);
```

Comment:
I'll try to repro again - but - they are not supposed to be visible... when do user see them? I can leave them alone, after all, there is no bug really,